.github/workflows/generate.yaml

```
@@ -0,0 +1,42 @@
+name: Generate API clients
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".openapi/**"
+
+jobs:
+  generate:
+    env:
+      GOPATH: /home/runner/work/app-services-sdk-go/go
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Generate API clients
+        run: yarn generate
+      - uses: peter-evans/create-pull-request@v3
+        with:
+          title: "chore(all): re-generate OpenAPI client(s)"
+          token: "${{ secrets.CI_USER_TOKEN }}"
+          commit-message: "chore(all): re-generate OpenAPI client(s)"
+          author: "app-services-ci <app-services-ci@users.noreply.github.com>"
+          branch: chore/generate-api-clients
+          branch-suffix: timestamp
+          reviewers: craicoverflow, wtrocki
+          delete-branch: true
```

```suggestion
          delete-branch: suffix
```

Actually that is a bug that I caused - fixed on Go SDK and forgot to check here. `true` will not work.

---

Ignore me, looking at the wrong property.